notebooks/official/migration/sdk-custom-scikit-learn-prebuilt-container.ipynb

```diff
@@ -1268,9 +1226,9 @@
         "id": "make_test_item:scilearn,tabular,census"
       },
       "source": [
-        "### Make test item\n",
+        "#### Create a test item\n",
         "\n",
-        "You use synthetic data as a test data item. Don't be concerned that we are using synthetic data -- we just want to demonstrate how to make a prediction."
+        "Use synthetic data as a test data item. Don’t be concerned about using synthetic data – as it's just for demonstration purpose"
```

Comment:
change "as" to "since" and "purpose" to "purposes"

 – since it's just for demonstration purposes